### ISSUE TITLE
Scene not loading in nbconvert's html

### ISSUE THREAD

**Issue by renefritze** [jupyter-widgets/pythreejs#285]
I'm trying to get a static representation of this [notebook](https://github.com/pymor/pymor/blob/py3js/notebooks/elliptic.ipynb) ([run on mybinder](https://mybinder.org/v2/gh/pymor/pymor/py3js?filepath=elliptic.ipynb)). I'm using the 2.1.1 release. 
If I run that through `jupyter nbconvert elliptic.ipynb` in the resulting page the two scenes do not load. The displaying colorbar is not a threejs scene, but just an image. I'm attaching the javascript console log. I've tried using the chrome debugger to stop in the offending line to see what's actually passed, but Chrome just stops somewhere else when I set a breakpoint. This is also the first time ever I've tried using a javascript debugger. 

[threejs-1568726241646.log](https://github.com/jupyter-widgets/pythreejs/files/3624848/threejs-1568726241646.log)

If in the same dockerized environment I run the `Geometries.ipynb` example through nbconvert those renders load fine from the static html. 

The relevant library code for the two plots is [here](https://github.com/pymor/pymor/blob/py3js/src/pymor/gui/jupyter/threejs.py#L69)

I hope you can help me figure out what's going wrong here.

**Comment by vidartf** [comment]
I don't really have the time to dig through the library code in detail, but from a quick scan I will note that `exec_three_obj_method` won't do anything when running in nbconvert. If you are able to make a *minimal* reproducible example using just pythreejs, that should make things a lot clearer for the both of us :)

**Comment by renefritze** [comment]
In trying to get a more minimal example, I realized that the whole asyncio setup I got is basically not executed when running through nbconvert. So I'm checking that out.

Could you maybe have a peek at the console log though? Maybe that error tells you something where I should be focussing on?

**Comment by renefritze** [comment]
The javascript error actually disappears when I ensure that the async loading is completed before nbconvert exits. So now I'm left with a white GL render which is brought on exactly by me using the `exec_three_obj_method` as somewhat of hack because my scene wouldn't display properly.

As such, I think the issue can be closed. Unless you think there is a way/case to try to generally translate the `exec_three_obj_method` to a static representation.

Either way, thank you for your help and a great library!

**Comment by vidartf** [comment]
> Unless you think there is a way/case to try to generally translate the `exec_three_obj_method` to a static representation.

Ideally, you shouldn't have to call that. There still might be some update flags that are not being set correctly by the controller. If you are able to get a pythreejs code only (as minimal as possible) that needs the update call to work, that would be useful.

**Comment by renefritze** [comment]
Getting a decoupled, minimal example of my issue proved difficult. 
However, I've finally managed to get rid of the `exec_three_obj_method` call by setting up the camera via an additional `cam.lookAt`. Which I promptly find to also use `exec_three_obj_method` and therefore being no help with nbconvert. What in the end actually fixed my camera snafu was simply translating the meshes to be centered on the origin, benefitting from the fact that camera is targeted there by default. 

In the end I've no idea what the real problem was. I tried coaxing the `Shadows` demo, which as a somewhat similar setup, into exhibiting the same camera problem I saw in my scene, but had there I never had an issue after nbconvert. Therefore I think this can be closed.
